src/brsTypes/components/RoSGNode.ts

```diff
@@ -1104,6 +1102,9 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
     }
 }
 
+// A node that represents the m.global, referenced by all other nodes
+export let mGlobal: RoSGNode = new RoSGNode([]);
```

Comment:
I had to move this down here because Typescript didn't want me to instantiate an `RoSGNode` above the class definition.